grammars/css.cson

```diff
@@ -336,176 +1214,367 @@
         'include': '#media-query'
       }
     ]
-  'numeric-values':
+  'media-types':
+    'captures':
+      '1':
+        'name': 'support.constant.media.css'
+      '2':
+        'name': 'invalid.deprecated.constant.media.css'
+    'match': '''(?xi)
+      (?<=^|\\s|,|\\*/)
+      (?:
+        # Valid media types
+        (all|print|screen|speech)
+        |
+        # Deprecated in Media Queries 4: http://dev.w3.org/csswg/mediaqueries/#media-types
+        (aural|braille|embossed|handheld|projection|tty|tv)
+      )
+      (?=$|[{,\\s;]|/\\*)
+    '''
+  'nested-selector':
```

Comment:
So while I would love nesting to be in CSS, it doesn't look like the relevant spec has been approved yet.  I'm not that keen on adding support for features that don't have support yet.